### PR TITLE
Remove stale classes and add more tests

### PR DESCRIPTION
**Pull request by nadeeshaan**
## Purpose
$title

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [x] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/rangeformat/RangeFormattingTest.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (pubudu91) [inline language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/rangeformat/RangeFormattingTest.java:2]: Is this a new test? Or moving an existing one?

Comment (nadeeshaan) [reply language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/rangeformat/RangeFormattingTest.java:2]: This is a new test. It's a mistake that the header contains an old year. I'll rectify this

Comment (nadeeshaan) [reply language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/rangeformat/RangeFormattingTest.java]: Fixed

Comment (nadeeshaan) [reply language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/rangeformat/RangeFormattingTest.java]: Fixed

**Review by IMS94 — APPROVED** [review]
LGTM!

**Pull request merged**